src/HtmlReader/HtmlReaderContent.tsx

```diff
@@ -4,22 +4,14 @@ const HtmlReaderContent: React.FC<{
   height: string;
   isScrolling: boolean;
   growsWhenScrolling: boolean;
-}> = ({ height, isScrolling, growsWhenScrolling }) => {
-  const shouldGrow = isScrolling && growsWhenScrolling;
+}> = ({ height }) => {
   return (
     <div id="D2Reader-Container">
       <main
         tabIndex={-1}
         id="iframe-wrapper"
         style={{
-          /**
-           * This determines the height of the iframe.
-           *
-           * If we remove this, then in scrolling mode it simply grows to fit
-           * content. In paginated mode, however, we must have this set because
-           * we have to decide how big the content should be.
-           */
-          height: shouldGrow ? 'initial' : height,
```

Comment:
Hang on, did we intend to remove this?

Reply:
We need to put this back actually. Maybe my comment wasn't clear what the use-case is for it, but this specifically affects growing-height readers. So go to the growing height embedded moby dick, and change it to scrolling mode. Then you will see that it does not actually grow in height anymore